ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/flow-tracking.yml

```diff
@@ -3,6 +3,8 @@
 
 flow_tracking:
   sampled:
+    encapsulation:
```

Comment:
can we try to exercise multiple options?

Reply:
added mpls as well